lib/generators/arclight/templates/catalog_controller.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 class CatalogController < ApplicationController
 
+  include BlacklightRangeLimit::ControllerOverride
```

Comment:
Do we need to add this in? Does the BlacklightRangeLimit add this automatically?